Project 7 - Bankist/js/user.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
@sknepali Not sure where we are using this json
If we are not using it, make sure you remove it.

Reply:
Ok sir